databricks/azure_auth_test.go

```diff
@@ -58,3 +54,10 @@ func TestAzureAuthCreateApiToken(t *testing.T) {
 
 	assert.NoError(t, instancePoolErr, instancePoolErr)
 }
+
+// getAndAssertEnv fetches the env for testing and also asserts that the env value is not Zero i.e ""
+func getAndAssertEnv(t *testing.T, key string) string {
+	value, present := os.LookupEnv("a")
+	assert.False(t, present, fmt.Sprintf("Env variable %s is not set", key))
+	return value
+}
```

Comment:
I believe this should be rather than what is currently there: 
```go
// getAndAssertEnv fetches the env for testing and also asserts that the env value is not Zero i.e ""
func getAndAssertEnv(t *testing.T, key string) string {
	value, present := os.LookupEnv(key)
	assert.True(t, present, fmt.Sprintf("Env variable %s is not set", key))
	return value
}
```

Reply:
Done